libcudacxx/include/cuda/std/__type_traits/always_false.h

```diff
@@ -0,0 +1,37 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of libcu++, the C++ Standard Library for your entire system,
+// under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+// SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES.
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef _LIBCUDACXX___TYPE_TRAITS_ALWAYS_FALSE_H
+#define _LIBCUDACXX___TYPE_TRAITS_ALWAYS_FALSE_H
+
+#include <cuda/std/detail/__config>
+
+#if defined(_CCCL_IMPLICIT_SYSTEM_HEADER_GCC)
+#  pragma GCC system_header
+#elif defined(_CCCL_IMPLICIT_SYSTEM_HEADER_CLANG)
+#  pragma clang system_header
+#elif defined(_CCCL_IMPLICIT_SYSTEM_HEADER_MSVC)
+#  pragma system_header
+#endif // no system header
+
+_LIBCUDACXX_BEGIN_NAMESPACE_STD
+
+template <class...>
+struct always_false : public false_type
+{};
+
+#if !defined(_CCCL_NO_VARIABLE_TEMPLATES)
+template <class... _Args>
+_CCCL_INLINE_VAR constexpr bool always_false_v = false;
+#endif // !_CCCL_NO_VARIABLE_TEMPLATES
```

Comment:
This should just be
```suggestion
template <class... _Args>
_CCCL_INLINE_VAR constexpr bool always_false_v = false;
```
We require C++17 now so variable templates are a given